pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[build-system]
+requires = [
+    "wheel",
+    "setuptools>=45",
+    "setuptools_scm[toml]>=6.2",
+]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+local_scheme = "dirty-tag"
+
+[tool.black]
+line-length = 88
+
+[tool.isort]
+profile = "black"
+multi_line_output = 3
+
+[tool.cibuildwheel]
+build-frontend = "build"
```

Comment:
We don't need this for the moment:

```suggestion
```

Reply:
Done in https://github.com/ami-iit/hippopt/pull/1/commits/7cff695da46883033771f59b12996a93269852d7